### ISSUE TITLE
`alpha` validator is not allowing unicode characters

### ISSUE THREAD

**Issue by josemotanet** [vuelidate/vuelidate#309]
Hello. I want to report a situation where the `alpha` validator is not fully abridging all possible characters.

## Steps to reproduce

* Use this fiddle: https://jsfiddle.net/josemota/8sxs1p2r/1/
* Type a non-ASCII string like `José` or `straße`.

Expected `$v.text.$valid` to be _true_. Actual value is **false**.

I suspect the regular expression to test against the value might be insufficient. [This StackOverflow question](https://stackoverflow.com/questions/150033/regular-expression-to-match-non-english-characters) might help.

**Comment by easynowbaby** [comment]
Seconded, does anyone have a solution for this?

**Comment by killemalljustice** [comment]
Any news on this?? :(

**Comment by tiagoppinho** [comment]
This is still an issue. Any news?

**Comment by mostafa-yasen** [comment]
I have the same issue, I think this can be fixed with a custom validator. I wish you can fix this to avoid wasting time.

**Comment by mostafa-yasen** [comment]
I've fixed this by customizing the `alpha` validator with the following regular expression

```javascript
import { alpha } from "vuelidate/lib/validators"
const alpha = helpers.regex("alpha", /^([^\u0000-\u007F]|[A-Za-z])+$/);
```

**Comment by lordpansar** [comment]
Would also be nice to have support for scandinavian characters, e.g Å, Ä, Ö, Ø, Æ etc.

**Comment by shentao** [comment]
@lordpansar Agree. I’d say all extended latin characters like śąęćżźó etc. Since we are now pretty close to getting a stable v2 (for Vue 3.0, but possibly also a compat version for Vue 2.x), maybe you could create a PR with a new `alphaExtended` validator?
Here are the new validators: https://github.com/vuelidate/vuelidate/tree/next/packages/validators

**Comment by dobromir-hristov** [comment]
We also have lots of cyrilic chars that are not covered. I wonder if there is a better alphanum regex/validation method.
